src/locale/locales/fr/messages.po

```diff
@@ -3314,7 +3314,7 @@ msgstr "Si vous supprimez cette liste, vous ne pourrez pas la récupérer."
 
 #: src/screens/Settings/components/ChangeHandleDialog.tsx:250
 msgid "If you have your own domain, you can use that as your handle. This lets you self-verify your identity. <0>Learn more here.</0>"
-msgstr ""
+msgstr "Si vous possédez votre propre domaine, vous pouvez l’utiliser comme pseudonyme. Cela vous permet d’auto-vérifier votre identité. <0>En savoir plus</0>"
```

Comment:
We've previously translated `handle` as simply `pseudo` in all the other strings, so I think it might make sense to stick with that here for consistency?

Also, for my own education, I was wondering: in French does it work better not to have a `.` after `En savoir plus` because it's a sentence fragment, rather than a complete sentence? 🤔

Reply:
Fixed ✅

And yeah, it's exactly for that reason I prefer not to keep "dangling full stops" at the end of such links. In my recollection, it's more common not to include them for `En savoir plus`, or to use ellipsis instead: `En savoir plus…` (but that's more of an old way of stating that _a dialog will follow_, as we used to do in UI copy in the old days!)

Reply:
Makes sense to me 👍